readme.md

```diff
@@ -248,7 +248,35 @@ Default: `'text'`
 
 The parsing method. Can be `'text'`, `'json'` or `'buffer'`.
 
-The promise also has `.text()`, `.json()` and `.buffer()` methods which sets this and the `resolveBodyOnly` option automatically.
+The promise also has `.text()`, `.json()` and `.buffer()` methods which return another Got promise returning the parsed body.\
+It's like setting the options to `{responseType, resolveBodyOnly: true}` but without affecting the main Got promise.\
```

Comment:
```suggestion
It's like setting the options to `{responseType: true, resolveBodyOnly: true}` but without affecting the main Got promise.

```

Reply:
> affecting the main Got promise

Affecting the main Got promise in what way? That needs to be clearer.

Reply:
> It's like setting the options to `{responseType: true

`responseType` is a string: `text` | `json` | `buffer`

> Affecting the main Got promise in what way? That needs to be clearer.

It doesn't change the request options. Instead, it clones the promise and instead of returning `response.body` it returns `JSON.parse(response.body)` for example.

Reply:
> It's like setting the options to `{responseType: true

`responseType` is a string: `text` | `json` | `buffer`

> Affecting the main Got promise in what way? That needs to be clearer.

It doesn't change the request options. Instead, it clones the promise and instead of returning `response.body` it returns `JSON.parse(response.body)` for example.

Reply:
Ok